models/nft/fantom/nft_fantom_transfers.sql

```diff
@@ -0,0 +1,96 @@
+{{ config(
+        alias ='transfers',
+        partition_by='block_date',
+        materialized='incremental',
+        file_format = 'delta',
+        unique_key = ['unique_transfer_id']
+)
+}}
+
+SELECT 'fantom'                                 AS blockchain
+     , t.evt_block_time                         AS block_time
+     , date_trunc('day', t.evt_block_time)      AS block_date
+     , t.evt_block_number                       AS block_number
+     , 'erc721'                                 AS token_standard
+     , 'single'                                 AS transfer_type
+     , t.evt_index
+     , t.contract_address
+     , t.tokenId                                AS token_id
+     , 1                                        AS amount
+     , t.from
+     , t.to
+     , t.evt_tx_hash                            AS tx_hash
+     , 'fantom' || t.evt_tx_hash || '-erc721-' || t.contract_address || '-' || t.tokenId || '-' || t.from || '-' ||
+       t.to || '-' || '1' || '-' || t.evt_index AS unique_transfer_id
+FROM {{ source('erc721_fantom', 'evt_transfer') }} t
+    {% if is_incremental() %}
```

Comment:
can we clean up this incremental block, to be just one, instead of two back-to-back? also, can we apply incremental date filters within the inner join, so both tables are reducing rows in the join?